MultiTarget/PackageReferences/Uno.props

```diff
@@ -1,24 +1,28 @@
 <Project>
+    <PropertyGroup>
+        <CommonUnoPackageVersion>4.9.45</CommonUnoPackageVersion>
```

Comment:
Why prefix with 'Common' maybe just 'UnoPlatformPackageVersion'?

Thinking also too that we may want to pull all these version numbers (like WinUI 2 and 3) and such into one of the other top-level files that's included everywhere in the chain, as I think we have a couple of spots where we've duplicated the WASDK version (sdk vs. tests I think)

Reply:
Uno does a good job of keeping their package versions in sync, but there's a few that don't do this (e.g. the wasm bootstrapper). Didn't want to cause confusion if we centralized these later.